Doc/library/subprocess.rst

```diff
@@ -95,6 +96,10 @@ compatibility with older versions, see the :ref:`call-function-trio` section.
 
       Added *encoding* and *errors* parameters
 
+   .. versionchanged:: 3.7
+
+      Added *text* parameter, as alias of *universal_newlines*
```

Comment:
Added the *text* parameter, as a more understandable alias of *universal_newlines*.